interface/reports/daily_summary_report.php

```diff
@@ -0,0 +1,409 @@
+                                                                    FROM `facility` AS f
+                                                                    LEFT JOIN `form_encounter` AS fe ON ( `fe`.`facility_id` = `f`.`id` )
+                                                                    LEFT JOIN `billing` AS b ON ( `fe`.`encounter` = `b`.`encounter` )
+                                                                    LEFT JOIN `users` AS u ON ( `fe`.`provider_id` = `u`.`id` )
+                                                                    WHERE `b`.`activity` =1 AND 
+                                                                    $whereTotalPaymentConditions
+                                                                    GROUP BY `b`.`encounter`,Date,provider_name ORDER BY Date ASC", $sqlBindArrayTotalPayment);
+
+        while ($totalPaymentRecord = sqlFetchArray($totalPaymetsSql)) {
+            $totalPayment[$totalPaymentRecord['Date']][$totalPaymentRecord['facilityName']][$totalPaymentRecord['provider_name']]['payments'] += $totalPaymentRecord['totalpayment'];
+        }
+
+        // total paid amount
+        $totalPaidAmountSql = sqlStatement("SELECT DATE( `p`.`dtime` ) AS Date,`f`.`name` AS facilityName, SUM( `p`.`amount1` ) AS totalPaidAmount, `fe`.`provider_id`, CONCAT( `u`.`fname`, ' ', `u`.`lname` ) AS provider_name
+                                                                        FROM `facility` AS f
+                                                                        LEFT JOIN `form_encounter` AS fe ON ( `fe`.`facility_id` = `f`.`id` )
+                                                                        LEFT JOIN `payments` AS p ON ( `fe`.`encounter` = `p`.`encounter` )
+                                                                        LEFT JOIN `users` AS u ON ( `fe`.`provider_id` = `u`.`id` )
+                                                                        WHERE $wherePaidConditions
+                                                                        GROUP BY `p`.`encounter`, Date,provider_name ORDER BY Date ASC", $sqlBindArrayPaid);
+
+
+        while ($totalPaidRecord = sqlFetchArray($totalPaidAmountSql)) {
+            $totalPaid[$totalPaidRecord['Date']][$totalPaidRecord['facilityName']][$totalPaidRecord['provider_name']]['paidAmount'] += $totalPaidRecord['totalPaidAmount'];
+        }
+
+        // merge all array recursive in to one array
+        $dailySummaryReport = array_merge_recursive($totalAppointment, $totalNewPatient, $totalVisit, $totalPayment, $totalPaid);
+        ?>
+
+        <div id="report_results" style="font-size: 12px">
+            <?php echo '<b>' . xlt('From', 'e') . '</b> ' . $from_date . ' <b>' . xlt('To', 'e') . '</b> ' . $to_date; ?>
```

Comment:
We have removed the e parameters from the xlt function.
Please see the changes at pull request https://github.com/bradymiller/openemr/pull/9